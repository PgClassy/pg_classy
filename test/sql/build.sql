\set ECHO none
BEGIN;
CREATE EXTENSION IF NOT EXISTS trunklet;

\i test/pgxntool/psql.sql

-- Need to create this manually
CREATE SCHEMA classy;

\echo
\echo INSTALL
\i sql/classy.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN
