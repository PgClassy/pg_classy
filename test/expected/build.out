\set ECHO none

INSTALL
 template__dependency__add 
---------------------------
 
(1 row)

 template__dependency__add 
---------------------------
 
(1 row)

 template__dependency__add 
---------------------------
 
(1 row)

# TRANSACTION INTENTIONALLY LEFT OPEN
